specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -239,7 +242,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/FileShare"
+              "$ref": "#/definitions/FileShareCreateParameters"
```

Comment:
Why are you forking the body definition for create/put? With the readonly fields of FileShare, FileShareCreateParameters is effectively the same. Doing this will generate a 2 classes the user would interact with for no real reason.

Reply:
EnabledProtocol is the property that can only be set at creation. We can stay with one body definition. But customer will be confused since enabledProtocol cannot be updated.